src/components/DraggableNode.tsx

```
@@ -0,0 +1,110 @@
+    onUpdate,
+    onPositionChange,
+    onDrag,
+    position
+}: Props) {
+    const [isEditing, setIsEditing] = React.useState(false)
+    const [showEditor, setShowEditor] = React.useState(false)
+    const nodeRef = useRef(null)
+
+    const sourceFiles = node.content.match(/source.*\.sh/g) || []
+
+    const handleDrag = (_: any, data: DraggableData) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)